src/Game.cpp

```diff
@@ -81,37 +81,36 @@ void main()
 
 	//char IP_address_send[NMAX_ADDRESS] = "2001:0:2877:7aa:3003:6f77:bd7c:618"; //Jeremy
 
-	//Jeremy: 2001:0:2877:7aa:3003:6f77:bd7c:618
-	//Nathan: 2001:0:2877:7aa:2cd4:6f77:476d:fceb
 
 	if (multiplayer)
 	{
-		port = 37000;//Socket 
+		//Activating network, socket for UDP6 com, and loading the initialization buffer to be sent while connecting players
+		port = 37000;//Setting port for networking
 		activate_network();
 		activate_socket6(port, IP_address_local, sock);
 		strcpy_s(buffer_init, "Connecting....");
-		size = 16;
+		size = 16;//Setting size for initial buffer while trying to connect
 		bool connected = false;
 		is_running = false;
 
-		while (!connected)
+		while (!connected)//Will continue to send messages until a message is received
 		{
-			send6(buffer_init, size, IP_address_send, sock, port);
+			send6(buffer_init, size, IP_address_send, sock, port);//sends init buffer
 			cout << "\n\nConnecting ...";
-			if (recv6(buffer_init, size, IP_address_recv, sock) > 0)
+			if (recv6(buffer_init, size, IP_address_recv, sock) > 0)//checks if any messages have been received
 			{
-				cout << "\nConnection secured from: " << IP_address_recv;
+				cout << "\nConnection secured from: " << IP_address_recv;//prints IPV6 of other player
 				connected = true;
 			}
 			Sleep(100);
 		}
-		if (connected)
+		if (connected)//Sends message once connected to ensure both players are connected
 		{
-			strcpy_s(buffer_init, "Connected!");
-			size = 11;
+			strcpy_s(buffer_init, "Connected!");//loading message to buffer after connection secured
+			size = 11;//Size of strcp_s message to be sent in buffer after connection secured
 			send6(buffer_init, size, IP_address_send, sock, port);
 			cout << "\n\nConnected";
-			is_running = true;
+			is_running = true;//starts game loop
 		}
 	}
```

Comment:
Here you send the message "Connected!" and then cout "Connected"..... we're never actually receiving the "connected!" message from the network. We should remove this entirely. And since we are using recv in a while loop, we will never move forward (start the game) without having a connection, so we can replace the connected bool with is_running = true at line 103.